lesson08/script.js

```diff
@@ -25,6 +25,7 @@ function createLists(value){
 }
 
 function SetDataToLists(){
+    createLoading();
     return new Promise((resolve,reject) => {
     const lists = [
         {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
```

Comment:
Change the createLoading function's position.
Considering "PROMISE", I think its position is certainly easy to understand.